Octree/include/CGAL/Octree/IO.h

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2007-2020  INRIA (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s)     : Jackson Campolattaro, Cédric Portaneri, Tong Zhao
+
+#ifndef CGAL_OCTREE_IO_H
+#define CGAL_OCTREE_IO_H
+
+#include <CGAL/license/Octree.h>
+
+#include <CGAL/Octree.h>
+#include <CGAL/Octree/Node.h>
+#include <CGAL/Octree/Walker_criterion.h>
+
+#include <iostream>
+#include <ostream>
+#include "Node.h"
+
+using std::ostream;
```

Comment:
Is that not a mistake, because it globally imports?

Reply:
Good catch!
https://github.com/JacksonCampolattaro/cgal/blob/7674b9f296734369e4e00ef38b11ae8af680d3c5/Octree/include/CGAL/Octree/IO.h#L15-L23
It should be resolved now, I still need to clean up my imports in general.